src/btouch.cs

```diff
@@ -224,8 +236,8 @@ static int Main2 (string [] args)
 					linkwith.Add (id);
 				}
 			},
-			{ "unified-full-profile", "Launches compiler pointing to XM Full Profile", l => { /* no-op*/ }, true },
```

Comment:
I think I have a better idea, we can extend our use of TargetFramework to use a TargetFrameworkMoniker: https://blogs.msdn.microsoft.com/visualstudio/2010/05/19/visual-studio-managed-multi-targeting-part-1-concepts-target-framework-moniker-target-framework/

So we'd end up with:

    --target-framework=Xamarin.Mac,Version=v4.5,Profile=Full
    --target-framework=Xamarin.Mac,Version=v4.5,Profile=System
    --target-framework=Xamarin.Mac,Version=v2.0,Profile=Mobile

as the possible arguments.